svt-av1-psy/PKGBUILD

```diff
@@ -4,8 +4,8 @@
 # Contributor: Zak BlueSwordM <neutronpcxt@gmail.com>
 
 pkgname=svt-av1-psy
-pkgver=v2.0.0.rc2.r0.g5cac47c1
-pkgrel=2
+pkgver=v2.0.0.r0.gfe919d9f
+pkgrel=3
```

Comment:
You dont need to bump the pkgrel on a git package

Reply:
It is fixed now. Put it back to 2.